tools/esdlcomp/esdlcomp.cpp

```diff
@@ -1109,22 +1109,20 @@ ESDLcompiler::ESDLcompiler(const char * sourceFile, bool generatefile, const cha
     StringBuffer prot;
     splitFilename(sourceFile, &prot, &srcDir, &name, &ext);
 
+    if (includePath_ && *includePath_)
+        includePath.set(includePath_);
+
     filename = strdup(sourceFile);
 
     yyin = fopen(sourceFile, "rt");
     if (!yyin)
     {
         if (isIncludedEsdl)
         {
-            StringBuffer alternateExtFilename;
-            alternateExtFilename.setf("%s%s%s", (prot.length()>0) ? prot.str() : "", srcDir.str(), name.str());
-
-            if (stricmp(ext.str(), ESDL_FILE_EXTENSION)==0)
-                alternateExtFilename.append(LEGACY_FILE_EXTENSION);
-            else
-                alternateExtFilename.append(ESDL_FILE_EXTENSION);
-
-            yyin = fopen(alternateExtFilename.str(), "rt");
+            StringBuffer includedFilePath;
```

Comment:
there's only a little 'd' between the names includeFilePath and includedFilePath.  Maybe this variable could be called "locatedFilePath"?